<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuTeeN</class>
 <widget class="QMainWindow" name="MuTeeN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MuTeeN Launcher 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource/resource.qrc">
    <normaloff>:/application.ico</normaloff>:/application.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
background:url(:/load.jpg);
}</string>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>418</y>
      <width>656</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>22</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border-radius: 5px;
    color:#ffffff;
}

QProgressBar::chunk {
    border-radius: 5px;
    background:rgb(0, 85, 127);
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>378</y>
      <width>447</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border-radius: 5px;
    color:#ffffff;
}

QProgressBar::chunk {
    border-radius: 5px;
    background:rgb(0, 85, 127);
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#ffffff;
color : white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource/resource.qrc">
      <normaloff>:/closeicon.jpg</normaloff>:/closeicon.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfiguracao">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>105</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>105</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>105</width>
      <height>28</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	margin:0;
	border :0;
	background: url(:/btn0.jpg);
	color: white;
}
QPushButton::hover
{
	margin:0;
	border :0;
	background: url(:/btn.jpg);
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Configurações</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartGame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>105</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>105</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>105</width>
      <height>28</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	margin:0;
	border :0;
	background: url(:/btn0.jpg);
	color: white;
}
QPushButton::hover
{
	margin:0;
	border :0;
	background: url(:/btn.jpg);
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Iniciar Jogo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Apreciamos sua escolha!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
